src/modules/nmx/geometry/Config.h

```diff
@@ -45,6 +45,9 @@ class Config : public VMM3Config {
     uint16_t MaxXSpan{10};
     uint16_t MaxYSpan{10};
     uint16_t DefaultMinADC{50};
+    bool SplitMultiEvents{false};
+    float SplitMultiEventsCoefficientLow{0.8};
```

Comment:
single tolerence parameter?

Reply:
Will bear in mind that this could be done more optimally - will wait on further requirements for multi-events before implementing anything, functionality might be entirely unnecessary